src/Layers/xrRenderPC_GL/r2_R_sun.cpp

```diff
@@ -325,7 +326,7 @@ struct DumbClipper
                     }
                 }
                 break;
-            } // switch (res)
+            }
```

Comment:
The switch is big enough, it's better to leave the comment.
```suggestion
            } // switch (res)
```